android/app/src/main/java/org/smartregister/fhircore/viewmodel/PatientListViewModel.kt

```diff
@@ -73,7 +73,12 @@ class PatientListViewModel(application: Application, private val fhirEngine: Fhi
     MutableLiveData<List<Immunization>>()
   }
 
-  fun searchResults(query: String? = null, page: Int = 0, pageSize: Int = 10) {
+  fun searchResults(
+    query: String? = null,
+    page: Int = 0,
+    pageSize: Int = 10,
+    showOverdue: Boolean = true
```

Comment:
I don't think we need to modify this fun signature

Reply:
https://github.com/opensrp/fhircore/pull/194/files#r645400196

Reply:
fixed. showOverdue param removed.